modules/logging-loki-log-access.adoc

```diff
@@ -74,16 +74,36 @@ subjects:
 // tag::CustomAdmin[]
 == Custom admin group access
 
-If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. Admin users have access to all application logs in all namespaces, if they also get assigned the `cluster-logging-application-view` role.
+// tag::LokiMode[]
+If you have a large deployment with several users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered administrators. 
```

Comment:
```suggestion
If you have a large deployment with several users who require broader permissions, you can create a custom group to configure with `adminGroup` field of LokiStack. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered administrators. 
```
I don't adminGroup field doesn't create group itself, a group has to be created prior before specifying in adminGroup field.

Reply:
@memodi This paragraph is for Loki docs. The following paragraph with the `/ tag::NetObservMode[]` is for NetObserv, and the piece you commented about is slightly different: 

> If you need to see cluster-wide logs without necessarily being an administrator, or if you already have any group defined that you want to use here, you can specify a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR have the same read access to logs as administrators.

Reply:
I see, understood, then ignore my comment.